launch/jdt.ls.remote.server.launch

```diff
@@ -13,7 +13,9 @@
 <booleanAttribute key="clearwslog" value="false"/>
 <stringAttribute key="configLocation" value="${workspace_loc}/.metadata/.plugins/org.eclipse.pde.core/jdt.ls.remote.server"/>
 <booleanAttribute key="default" value="false"/>
-<stringAttribute key="deselected_workspace_plugins" value="org.eclipse.jdt.ls.tests,org.eclipse.jdt.ls.tests.syntaxserver,org.eclipse.jdt.ui"/>
+<setAttribute key="deselected_workspace_bundles">
+<setEntry value="org.eclipse.jdt.ls.tests"/>
```

Comment:
deselect org.eclipse.jdt.ls.tests.syntaxserver too. 

Same for other *.launch files.

Reply:
and what about org.eclipse.jdt.ui?

Reply:
Fixed.